plugins/tds61_rules.js

```diff
@@ -4491,6 +4481,32 @@ tds61.rules = {
     ],
     // ##### End of shopList #####
 
+    // ##### Start of fuzzyTable #####
+    // This list uses the new IsA, IsSimilar etc functions to map a number of input values to a single output
+    fuzzyTable : [
+     ['ZI016_ROC','7',schemaTools.isA('surface=paved', 0.8, 0.1, 0.5)], // Ridgid Pavement
+     ['ZI016_ROC','3',schemaTools.isA('surface=unpaved', 0.8, 0.1, 0.5)], // Flexible Pavement
+
+     ['ZI016_ROC','1',schemaTools.simple('surface=ground',2)], // Unimproved
+//      ['ZI016_ROC','2',schemaTools.simple('surface=compacted',2)], // Stabilized earth
+     ['ZI016_ROC','4',schemaTools.simple('surface=gravel',2)], // Aggregate
+//      ['ZI016_ROC','5',schemaTools.simple('surface=macadam',2)], // Macadam
+//      ['ZI016_ROC','6',schemaTools.simple('surface=bound_surface',2)], // Bound Surface
+     ['ZI016_ROC','8',schemaTools.simple('surface=concrete',2)], // Concrete
+     ['ZI016_ROC','9',schemaTools.simple('surface=asphalt',2)], // Asphalt
+//      ['ZI016_ROC','10',schemaTools.simple('surface=asphalt_over_concrete',2)], // Asphalt over Concrete
+     ['ZI016_ROC','11',schemaTools.simple('surface=cobblestone',2)], // Cebble-stone
```

Comment:
@mattjdnv Should this be? This will capture cobblestone:flattened too.

['ZI016_ROC','11',schemaTools.isA('surface=cobblestone',1)],